kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -0,0 +1,357 @@
+
+def kieServerCapabilitiesMarker = 'KIE_SERVER_CAPABILITIES_MARKER';
+def jbpmConfigMarker = 'JBPM_CONFIG_MARKER';
+def databaseConfigMarker = 'DATABASE_CONFIG_MARKER';
+def databaseDependsMarker = 'DB_DEPENDS_MARKER';
+def springBootStarterMarker = 'KIE_SPRING_BOOT_STARTER_MARKER';
+def indexCSSMarkerBA = "INDEX_CSS_MARKER_BA";
+def indexIconMarkerBA = "INDEX_ICON_MARKER_BA";
+def indexCSSMarkerDM = "INDEX_CSS_MARKER_DM";
+def indexIconMarkerDM = "INDEX_ICON_MARKER_DM";
+def indexCSSMarkerBO = "INDEX_CSS_MARKER_BO";
+def indexIconMarkerBO = "INDEX_ICON_MARKER_BO";
+def myServiceNameMarker = 'MYSERVICE_NAME_MARKER';
+def myServiceVersionMarker = 'MYSERVICE_VERSION_MARKER';
+
+def BPMSpringBootStarterDepends = """
+<dependency>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-server-spring-boot-starter</artifactId>
+    <version>\${version.org.kie}</version>
+</dependency>
+""";
+
+def BRMSpringBootStarterDepends = """
+<dependency>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-server-spring-boot-starter-drools</artifactId>
+    <version>\${version.org.kie}</version>
+</dependency>
+""";
+
+def PlannerSpringBootStarterDepends = """
+<dependency>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-server-spring-boot-starter-optaplanner</artifactId>
+    <version>\${version.org.kie}</version>
+</dependency>
+""";
+
+def h2Depends = """
+<dependency>
+  <groupId>com.h2database</groupId>
+  <artifactId>h2</artifactId>
+</dependency>
+""";
+
+def mysqlDepends = """
+<dependency>
+  <groupId>mysql</groupId>
+  <artifactId>mysql-connector-java</artifactId>
+</dependency>
+<dependency>
+  <groupId>com.h2database</groupId>
+  <artifactId>h2</artifactId>
+</dependency>
+""";
+
+def postgreslDepends = """
+<dependency>
+  <groupId>org.postgresql</groupId>
+  <artifactId>postgresql</artifactId>
+</dependency>
+<dependency>
+  <groupId>com.h2database</groupId>
+  <artifactId>h2</artifactId>
+</dependency>
+""";
+
+def serverCapabilitiesBPM = """
+#kie server capabilities
+kieserver.drools.enabled=true
+kieserver.dmn.enabled=true
+kieserver.jbpm.enabled=true
+kieserver.jbpmui.enabled=true
+kieserver.casemgmt.enabled=true
+""";
+
+def serverCapabilitiesBRM = """
+#kie server capabilities
+kieserver.drools.enabled=true
+kieserver.dmn.enabled=true
+""";
+
+def serverCapabilitiesPlanner = """
+#kie server capabilities
+kieserver.drools.enabled=true
+kieserver.dmn.enabled=true
+kieserver.optaplanner.enabled=true
+""";
+
+def BPMJBPMConfig = """
+#jbpm configuration
+jbpm.executor.enabled=false
+#jbpm.executor.retries=5
+#jbpm.executor.interval=3
+#jbpm.executor.threadPoolSize=1
+#jbpm.executor.timeUnit=SECONDS
+""";
+
+def BPMDatabaseConfigH2 = """
+#data source configuration
+spring.datasource.username=sa
+spring.datasource.password=sa
+spring.datasource.url=jdbc:h2:./target/spring-boot-jbpm;MVCC=true
+spring.datasource.driver-class-name=org.h2.Driver
+spring.datasource.type=org.apache.tomcat.jdbc.pool.XADataSource
```

Comment:
this data source type should be removed, it caused problems when moving to newer version of spring boot and it's not really needed